<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogBase</class>
 <widget class="QDialog" name="CDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="m_movieLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="m_frameSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="m_openButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_playButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pauseButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_stopButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_quitButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
